src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/AsyncActionPlanEventService.kt

```diff
@@ -25,7 +25,7 @@ class AsyncActionPlanEventService(
       log.info { "ActionPlan created event for prisoner [${actionPlan.prisonNumber}]" }
       launch {
         actionPlan.goals.forEach {
-          telemetryService.trackGoalCreateEvent(it)
+          telemetryService.trackGoalCreatedEvent(it)
```

Comment:
As wel as renaming the actual events that are sent to App Insights. I've renamed the corresponding methods on the service class